api/src/upload-default-docs.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('underscore');
+const fs = require('fs');
+const path = require('path');
+
+const environment = require('./environment');
+const settingService = require('./services/settings');
+const uploadDocumentsInFolder = require('./upload-documents-in-folder');
+
+const uploadInitialConfig = async function () {
+  const canFetchConfiguration = () => settingService.get().then(settings => !_.isEmpty(settings)).catch(() => false);
+  const hasConfiguration = await canFetchConfiguration();
```

Comment:
I think it'd be clearer to inline this function. Also instead of requiring underscore I'd probably just use `Object.keys`, eg:

```js
const hasConfiguration = await settingService.get()
  .then(settings => Object.keys(settings).length)
  .catch(() => false);
```